<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receive</class>
 <widget class="QMainWindow" name="receive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>999</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>104</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>服务请求</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>104</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>支付通知</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>999</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>411</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>订单显示</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
